tests/filtering/test_filtering.py

```diff
@@ -3,8 +3,11 @@
 import pandas as pd
 import numpy as np
 
-from moonstone.filters.filtering import (
-    Filtering
+from moonstone.filtering.filtering import (
+    Filtering,
+)
+from moonstone.filtering.mean_filtering import (
```

Comment:
I am being a bit picky here but I think you should create a dedicated module called `test_mean_filtering.py` to try to respect as much the architecture we have also in the code.